src/components/organisms/PaneManager/PaneManagerSplitView.tsx

```diff
@@ -88,7 +88,12 @@ const PaneManagerSplitView: React.FC = () => {
     }
 
     if (!leftActive) {
-      dispatch(setPaneConfiguration({...paneConfiguration, navWidth: navPaneWidth / splitViewContainerWidth}));
+      dispatch(
+        setPaneConfiguration({
+          ...paneConfiguration,
+          navWidth: splitViewContainerWidth === 0 ? 0 : navPaneWidth / splitViewContainerWidth,
```

Comment:
Still trying to figure out if `0` is the correct value or whether it needs to be simply current navPaneWidth. I'll try playing around with Monokle to see what looks best.

Reply:
It used to default to navPaneWidth and this problem was likely introduced in fd52da8faf2b5a6d4e14c13ac15f233e2c4abc0c. I'm reverting it to the code there that handled this null division.